### DISCUSSION TITLE
Laravel Nova: Group form fields in columns

### DISCUSSION THREAD

**Discussion by newtec-foggia** [laravel/nova-issues#4534] [Q&A]
How can i create columns on my form fields in Laravel Nova?

Does anyone have an idea how to solve it? Or has a package that does it? I already searched but nothing found.

Thanks

**Comment by m-lotze** [comment]
Use https://github.com/Formfeed-UK/nova-theming-classes and then you can create a grid with tailwind:

```css
.component-form-panel > .component-card {
    @apply grid grid-cols-12;

    .component-field-wrapper {
        @apply col-span-12;
    }

    .field-a {
        @apply col-span-2;
    }

    .field-b {
        @apply col-span-3;
    }

    ....
}
```

**Reply by newtec-foggia** [reply]
sorry, it is not clear to me, in the resource I have these fields, once the package is installed how do I proceed?

![fields](https://user-images.githubusercontent.com/83398776/177731677-adfc5c97-2560-4a4f-9e7f-7f1c00b423ee.JPG)

Thanks

**Reply by m-lotze** [reply]
You need to add a stylesheet to nova, e.g. with `Nova::style(...path_to_css..)` in NovaServiceProvider with the CSS for the grid. Either tailwind or use normal CSS.

Tailwind docs for grid: https://tailwindcss.com/docs/grid-template-columns
